lib/kitchen/directions/bake_chapter_references/v2.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+# The main difference between v1 is that in v2 references are also baked form introduction pages
+module Kitchen::Directions::BakeChapterReferences
+  class V2
+    renderable
+
+    def bake(chapter:, metadata_source:, uuid_prefix: '.', klass: 'references')
+      @metadata = metadata_source.children_to_keep.copy
+      @klass = klass
+      @title = I18n.t(:references)
+      @uuid_prefix = uuid_prefix
+
+      chapter.references.search('h3').trash
```

Comment:
is this line redundant with lines 19 & 31?

Reply:
@kswanson33 the line was already there. Hmm maybe it is a leftover or we should keep it just for safety reasons? What do you think?

Reply:
@LostMojo through the lens I have found out that you added the line from above? Could you advise if it is still needed form some reason if you still remember off course :upside_down_face:

Reply:
If I remember correctly, the one on line 13 was to catch h3's in references that didn't fall into a non-introduction page.

Reply:
Oh, Thank you MaryJo. :relaxed:

Reply:
@LostMojo can I have just one more question regarding references in your book because as I understood from your answer from above is that you needed to remove the h3 titles from references from intro pages but you were not creating a k\link title for them, is that the case?

Reply:
They weren't in the introduction pages, they were composite pages (that don't get picked up as real pages, sometimes). I don't... think I was creating a k/link title for them, though :\